### DISCUSSION TITLE
Issue나 Pull request에 대한 정해진 템플릿이 있었으면 좋겠습니다.

### DISCUSSION THREAD

**Discussion by falconlee236** [manito42/FrontEnd#33] [General]
issue나 pull request에는 정해진 템플릿이 없는건가요??
없으면  템플릿 만드는 것에 대해서 시간 남을때 한번 다같이 의논해봐도 좋을것 같습니다. 

방금 이슈를 작성하면서 다른 사람들이 이슈를 올리는 것을 참고하면서 작성을 일단 하긴 했는데, 
정해진 양식이 없는 것 같아서 한번 올려봅니다.

정해진 양식이 있으면 보는 사람도 무슨 내용인지 쉽게 알 수 있고, 작성하는 사람들도 빠르게 틀에 맞춰서 작성하면 되기 때문에
효율이 좋아질 것 같습니다

**Comment by JuneParkCode** [comment]
좋은 의견이라고 생각합니다. 템플릿 찾아보도록하겠습니다!

**Reply by falconlee236** [reply]
답변 감사합니다!! 저도 같이 찾아보고 괜찮은 것 있으면 공유할게요
